### PR TITLE
Cleanup iostats field naming

### PR DESCRIPTION
**Pull request by ruflin**
Align naming with other metricsets.

### CODE REVIEW

File: metricbeat/docs/fields.asciidoc [hunk 1]
```diff
@@ -8773,63 +8773,67 @@ The total number of of milliseconds spent doing I/Os.
...
+=== `system.diskio.iostat.write.request.per_sec`
 
 type: float
 
 The number of write requests that were issued to the device per second
 
 
 [float]
-=== `system.diskio.iostat.read_byte_per_sec`
+=== `system.diskio.iostat.read.per_sec.bytes`
```
Comment (monicasarbu) [inline metricbeat/docs/fields.asciidoc]: what about `read.bytes.per_sec` to be as the previous one?

File: metricbeat/module/system/diskio/diskio.go [hunk 2]
```diff
@@ -65,17 +65,33 @@ func (m *MetricSet) Fetch() ([]common.MapStr, error) {
...
-				"read_byte_per_sec":            extraMetrics.ReadBytesPerSec,
-				"write_byte_per_sec":           extraMetrics.WriteBytesPerSec,
-				"avg_request_size":             extraMetrics.AvgRequestSize,
-				"avg_queue_size":               extraMetrics.AvgQueueSize,
-				"await":                        extraMetrics.AvgAwaitTime,
-				"service_time":                 extraMetrics.AvgServiceTime,
-				"busy":                         extraMetrics.BusyPct,
+				"read": common.MapStr{
+					"request": common.MapStr{
+						"merge_per_sec": extraMetrics.ReadRequestMergeCountPerSec,
```
Comment (monicasarbu) [inline metricbeat/module/system/diskio/diskio.go]: Shouldn't be `merged_per_sec` as in the description of the `fields.yml`?

**Comment by monicasarbu** [comment]
Looks good to me. I left a few minor comments.

File: metricbeat/docs/fields.asciidoc (hunk 1 above)
Comment (ruflin) [inline metricbeat/docs/fields.asciidoc]: I put `byte` at the end as this is the unit.

File: metricbeat/module/system/diskio/diskio.go (hunk 2 above)
Comment (ruflin) [inline metricbeat/module/system/diskio/diskio.go]: Changed it in both places to `merges_per_sec`

**Pull request merged**